client/login/controller.js

```diff
@@ -16,6 +17,25 @@ const enhanceContextWithLogin = ( context ) => {
 		query,
 	} = context;
 
+	// Process a social login handoff from /start/user.
+	if ( query?.email_address && query?.source && query?.access_token && query?.id_token ) {
+		context.store.dispatch( {
```

Comment:
@p-jackson Oddly, when I'm running this code at `http://calypso.localhost:3000/`, `context.store.dispatch()` successfully sets the store. However, when I'm running the code at `https://wpcalypso.wordpress.com/`, it doesn't set the store. Any ideas why this might be the case?

Reply:
Is it just a typo thing? During testing the debugger wasn't stepping into the branch of the code that dispatched the action. But I changed `query?.source` to `query?.service` and it worked, and I think `service` is what we want anyway.

Reply:
> Is it just a typo thing? During testing the debugger wasn't stepping into the branch of the code that dispatched the action. But I changed `query?.source` to `query?.service` and it worked, and I think `service` is what we want anyway.

D'oh!